l10n_it_website_sale_fiscalcode/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Nicola Malcontenti - Agile Business Group
```

Comment:
Replace `©` by `Copyright` everywhere according to
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2